app/assets/javascripts/event-stage.js

```diff
@@ -478,7 +477,6 @@
                         return v.toString(16);
                     });
                     eventStage.data.eventModel.slug = uuid;
```

Comment:
I think this line is a problem. I don't understand why we would ever set an event slug to a uuid.